checker/build.gradle

```diff
@@ -221,6 +221,7 @@ clean {
 }
 
 clean.doLast {
+  buildDir.deleteDir()
```

Comment:
This change should probably be in a separate PR

Reply:
Fixed. Thanks!